go.mod

```diff
@@ -5,7 +5,7 @@ go 1.14
 require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/giantswarm/apiextensions/v2 v2.5.1
-	github.com/giantswarm/certs/v3 v3.0.0
+	github.com/giantswarm/certs/v3 v3.0.1-0.20200925114212-d08b0085d6b0
```

Comment:
This should be properly tagged before merging.

Reply:
It will be changed after releasing `certs` :+1: